### PR TITLE
Changing ID types to Byte

### PR DESCRIPTION
**Pull request by carlosvdr**
issue #17 
Based on this [recommendations](https://medium.com/edge-node-engineering/two-simple-subgraph-performance-improvements-a76c6b3e7eac)

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -20,25 +20,25 @@ type Sender @entity {
 
 type Receiver @entity {
   # receiver address
-  id: ID!
+  id: Bytes!
   escrowAccounts: [EscrowAccount!] @derivedFrom(field: "receiver")
   transactions: [Transaction!] @derivedFrom(field: "receiver")
 }
 
 type Transaction @entity{
```
Comment (aasseman) [inline schema.graphql:28]: From the recommendations, you can also set `Transaction` as an immutable entity.

**Review by aasseman — CHANGES_REQUESTED** [review]

**Comment by aasseman** [comment]
One thing to note is that the IDs built with `concat` are different than before (breaking change). Though that shouldn't break anything in the TAP software as we don't use those IDs there.

File: schema.graphql (hunk 1 above)
Comment (carlosvdr) [reply schema.graphql:28]: Oh you are right! I did read it and thought none was inmutable but yeah transaction is

**Review by aasseman — APPROVED** [review]
LGTM

**Review by aasseman — APPROVED** [review]

**Comment by aasseman** [comment]
You'll need to sign your commits to make the checks happy

**Pull request merged**